### PR TITLE
Fix checks redirect in 2.15 docs

### PR DESCRIPTION
**Pull request by travisbeckham**

### CODE REVIEW

File: linkerd.io/content/2.15/checks/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <script type="text/javascript">
     window.onload = function() {
       var hash = window.location.hash;
-      window.location.href = window.location.origin + "/2.14/tasks/troubleshooting/" + hash;
+      window.location.href = window.location.origin + "/2.15/tasks/troubleshooting/" + hash;
```
Comment (wmorgan) [inline linkerd.io/content/2.15/checks/index.html:9]: Would it make sense to have this be `/2/tasks/troubleshooting/`?

Comment (travisbeckham) [reply linkerd.io/content/2.15/checks/index.html:9]: Yeah. That should be fine. I should probably update all of the old docs to do the same.

**Review by wmorgan — COMMENTED** [review]
Thanks! Since this is a new type of error I would advocate for adding some new language to the help doc as well. I will push a commit to the branch.

**Review by wmorgan — APPROVED** [review]
Looks great, thank you

**Pull request merged**